packages/test-utils/src/wrapper.js

```diff
@@ -667,7 +668,7 @@ export default class Wrapper implements BaseWrapper {
     })
 
     // $FlowIgnore : Problem with possibly null this.vm
-    this.vnode = this.vm._vnode
+    this._vnode = this.vm._vnode
```

Comment:
@eddyerburgh 
There is a problem.
This line did not make error at #748.
Is this line necessary?

Reply:
You can flowignore it

Reply:
I removed this line.

Reply:
Since vueWrapper.vnode is read-only, I removed this line.